src/rpc/util.cpp

```diff
@@ -830,16 +830,15 @@ void RPCResult::ToSections(Sections& sections, const OuterType outer_type, const
         return;
     }
     case Type::OBJ_DYN:
```

Comment:
I guess the breakage was in `OBJ_DYN` falling through to `OBJ_EMPTY` instead of `OBJ`?

Reply:
Yes